jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/verification/RegexFindVerifier.java

```diff
@@ -0,0 +1,22 @@
+package org.jbehavesupport.core.internal.verification;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.springframework.stereotype.Component;
+import org.springframework.util.Assert;
+import static org.jbehavesupport.core.internal.verification.VerifierNames.REGEX_FIND;
+
+@Component
+public class RegexFindVerifier extends AbstractVerifier {
```

Comment:
Can you please add similar test as for other verifiers?

Reply:
I used test for RegexVerifier (REGEX_MATCH) and added teo more cases to better fit this verifier.